app_base.htm

```diff
@@ -0,0 +1,83 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="navbar">
+<header>
+
+    <nav class="navbar navbar-inverse">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <img height="50" id="menu-img" src="${ STATIC_URL }homepage/media/images/musicnote.jpg"
+                     alt="Music Note">
+            </div>
+            <ul class="nav navbar-nav">
+                <li class="${ 'active' if request.dmp.page == 'homepage/index' else ' ' }"><a href="/homepage/index">Home</a>
+                </li>
+                <li class="${ 'active' if request.dmp.page == 'about' else ' ' }"><a href="/homepage/about">About</a>
+                </li>
+                <li class="${ 'active' if request.dmp.page == 'contact' else ' ' }"><a
+                        href="/homepage/contact">Contact</a></li>
+                <li class="${ 'active' if request.dmp.page == 'FAQ' else ' ' }"><a href="/homepage/FAQ">FAQ</a></li>
+                <li class="${ 'active' if request.dmp.page == 'terms' else ' ' }"><a href="/homepage/terms">Terms</a>

```

Comment:
Silly Blake, make your code more readable by pressing enter between elements like the nested a tag in your li tags.

Reply:
I was under the impression that excessive blank line spacing can eventually slow a system. Is that not true?